typescript/libs/service_proxy.py

```diff
@@ -294,6 +294,24 @@ def quick_info(self, path, location=Location(1, 1), on_completed=None):
                 req_dict["seq"]
             )
 
+    def quick_info_full(self, path, location=Location(1, 1), on_completed=None):
```

Comment:
Does it make sense to make this `on_completed = lambda: None`?

Reply:
This function is essentially just a copy paste of the non-full version above it. Seemed reasonable to keep the two functions as close to the same as posisble.